flink-autoscaler-standalone/src/test/java/org/apache/flink/autoscaler/standalone/flinkcluster/FlinkClusterJobListFetcherTest.java

```diff
@@ -231,6 +214,22 @@ CompletableFuture<P> sendRequest(M h, U p, R r) {
                             return (CompletableFuture<P>)
                                     CompletableFuture.completedFuture(
                                             configurationsOrException.left().get(jobID));
+                        } else if (h instanceof JobsOverviewHeaders) {
```

Comment:
Changing this file due to we start request `JobsOverviewHeaders` instead of `listJobs` in this PR.